src/main/java/com/example/simplefileshare/simplefileshare/data/Filedata.java

```diff
@@ -0,0 +1,17 @@
+package com.example.simplefileshare.simplefileshare.data;
+
+
+public class Filedata {
```

Comment:
We can use Lombok to automatically generate getters and setters

Reply:
Okay. Will check it.